docs/Users/Install.rst

```diff
@@ -6,12 +6,12 @@ platforms are Linux and Windows. *coala* is known to work on OS X as well.
 *coala* is tested against CPython 3.3, 3.4 and 3.5.
 
 In order to run *coala* you need to install Python. It is recommended,
-that you install Python3 >= 3.3 from http://www.python.org.
+that you install Python3 >= 3.3 from `here <https://www.python.org/downloads/>`_.
 
 The easiest way to install *coala* is using pip (Pip Installs Packages).
-If you don't already have pip, you can install it like described on
-https://pip.pypa.io/en/stable/installing.html. Note that pip is shipped
-with recent python versions by default.
+If you don't already have pip, you can install it like described in the
+`pip installation guide <https://pip.pypa.io/en/stable/installing.html>`_. Note that
+pip is shipped with recent python versions by default.
```

Comment:
now that you mentioned this, you can also say: To check if you have pip already installed, use ``pip show pip`` and it will tell you what version is installed.